src/items/themes/giantblood.json

```diff
@@ -11,7 +11,7 @@
     "type": "",
     "modifiers": [],
     "abilityMod": {
-      "ability": "str",
```

Comment:
Maybe there's something I'm not understanding here but the old value looks correct to me
![image](https://user-images.githubusercontent.com/2079751/99939517-810f9f00-2d6a-11eb-9895-8b23a5e1c07e.png)

Reply:
Whoops That was a mistake. Can you look over merc I know it was comepletely wrong in the original file as well as priest had the wrong stat mod

Reply:
Wait. In Foundry when you look at the Giantblood theme is Says +1 Con which is where the initial mistake came from.
![Giantblood](https://user-images.githubusercontent.com/3371810/100032475-b98c9880-2dbd-11eb-9271-1297a1c8d22a.png)